src/operation_latency_histogram.cpp

```diff
@@ -19,6 +19,10 @@ constexpr double MIN_GLOB_LATENCY_MILLISEC = 0;
 constexpr double MAX_GLOB_LATENCY_MILLISEC = 1000;
 constexpr int GLOB_LATENCY_NUM_BKT = 100;
 
+constexpr double MIN_LIST_LATENCY_MILLISEC = 0;
+constexpr double MAX_LIST_LATENCY_MILLISEC = 1000;
```

Comment:
You should adjust latency outlier threshold, list usually takes longer than open and read.